Makefile

```diff
@@ -29,7 +29,7 @@ build:
 		rm -rf node_modules; \
 	fi
 	@which corepack > /dev/null || (echo "Installing corepack..." && npm install -g corepack)
-	@cd frontend && corepack enable && pnpm install && pnpm run make-i18n
+	@cd frontend && sudo corepack enable && pnpm install && pnpm run make-i18n
```

Comment:
🤔 this could cause issues in other environments. I generally try to avoid putting `sudo` inside a script like this

Does running `sudo make build` work? Any other options here?